File: practice_4-18/prac.py
def get_input():
    user_input = input("Please enter a number:")
    return int(user_input)


def compare_numbers(numOne, numTwo):
    # i = 1
    # while (i < numTwo + 1):
    #     product = numOne * i
    #     print(str(numOne) + " x " + str(i) + " = " + str(product))
    #     i += 1
    for i in range(1, numTwo + 1):
        product = numOne * i
        print(str(numOne) + " x " + str(i) + " = " + str(product))
    print("Done")


def main():
    compare_numbers(get_input(), get_input())


if __name__ == "__main__":
    main()

File: practice_4-18/Prac.java
import java.util.Scanner;

class Prac {
    public static void main(String[] args) {
        Scanner userInput = new Scanner(System.in);
        compareInts(userInput.nextInt(), userInput.nextInt());
        userInput.close();
    }

    public static void compareInts(int numOne, int numTwo) {
        // int counter = 1;
        // while (counter < numTwo + 1) {
        // int product = numOne * counter;
        // System.out.println(numOne + " x " + counter + " = " + product);
        // counter++;
        // }
        for (int i = 1; i < numTwo + 1; i++) {
            int product = numOne * i;
            System.out.println(numOne + " x " + i + " = " + product);
        }
    }
}
